Backend/dev/migrations/dynamic-offer-driver-app/0141-add-get-nearby-api.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE atlas_driver_offer_bpp.merchant_service_usage_config ADD COLUMN get_near_by_search character varying(30);
+UPDATE atlas_driver_offer_bpp.merchant_service_usage_config SET get_near_by_search = 'Google';
+ALTER TABLE atlas_driver_offer_bpp.search_request_location ADD COLUMN is_near_by_search_a_p_i_used boolean DEFAULT false NOT NULL;
```

Comment:
I think its better to name this column` is_near_by_search_api_used` and `isNearBySearchAPIUsed` in domain